test/stateless/consumed-proxy-code-cache.test.ts

```
@@ -0,0 +1,110 @@
+    // Exactly at expiry boundary — still rejected while expiresAt > now.
+    now += 999;
+    assert.equal(cache.tryReserve("hash-a", ttlSeconds), false);
+
+    // After TTL — purged; a new reservation is allowed (sealed code itself
+    // would also be expired by openStoredTokensCode at this point).
+    now += 1;
+    assert.equal(cache.tryReserve("hash-a", ttlSeconds), true);
+  });
+
+  test("never LRU-evicts a live entry; fails closed when full", () => {
+    let now = 5_000_000;
```

ESLint

[error] 'now' is never reassigned. Use 'const' instead.

(prefer-const)